tests/ios/xcodeproj/BUILD.bazel

```diff
@@ -11,6 +11,18 @@ xcodeproj(
     ],
 )
 
+xcodeproj(
+    name = "Test-Imports-App-Project",
+    testonly = True,
+    bazel_path = "bazelisk",
+    include_transitive_targets = False,
```

Comment:
Current post-build check will fail on `include_transitive_targets` being set to `True` because there are search paths are not found. I am planning to make this into an issue under github and deal with it later in a separate PR, what do you folks think? @segiddins @amberdixon

Reply:
if it's not a regression, feel free to make an issue for it and fast-follow